### ISSUE TITLE
Move API limits from per min to per hour 

### ISSUE THREAD

**Issue by azenMatt** [ZenHubIO/API#51]

**Comment by nickbaldwin** [comment]
Yep... this is killing me... given there is no endpoint to return a paged list of item (with a 'since' header/query) I have to retrieve all items... and then it takes several minutes!
